core/src/main/java/org/fao/geonet/MergeUsersByUsernameDatabaseMigration.java

```diff
@@ -103,6 +104,15 @@ private void transferSavedSelections(ApplicationContext applicationContext, List
         }
     }
 
+    private void transferMetadataStatus(ApplicationContext applicationContext, List<User> duplicatedUserList, User userToKeep) {
+        MetadataStatusRepository metadataStatusRepository = applicationContext.getBean(MetadataStatusRepository.class);
+        // TODO
```

Comment:
Can be removed the `TODO`?

Reply:
You mean simply remove it, or do what is expected ? There are other TODO in that file. Should I also implement all the missing stuff there ?

Reply:
Renamed methods and removed TODOs

Reply:
This method was new, I had no clear with the `TODO` what was missing. If there's any pending stuff, please just add a comment. 

I only refer to this method as seem new.

Reply:
It was just a copy/paste from an existing method supposed to transfer stuff but it was not and a TODO was in there. So it is probably better to simply rename those methods to delete stuff which in this case is not really important. I did it. Please check and merge if fine for you.